mimic/model/rackspace_image_store.py

```diff
@@ -0,0 +1,60 @@
+"""
+Image store for Rackspace images
+"""
+from characteristic import attributes, Attribute
+from rackspace_images import (RackspaceWindowsImage, RackspaceArchImage, RackspaceCentOSPVImage,
+                              RackspaceCentOSPVHMImage, RackspaceCoreOSImage, RackspaceDebianImage,
+                              RackspaceFedoraImage, RackspaceFreeBSDImage, RackspaceGentooImage,
+                              RackspaceOpenSUSEImage, RackspaceRedHatPVImage, RackspaceRedHatPVHMImage,
+                              RackspaceUbuntuPVImage, RackspaceUbuntuPVHMImage, RackspaceVyattaImage,
+                              RackspaceScientificImage, RackspaceOnMetalCentOSImage,
+                              RackspaceOnMetalCoreOSImage, RackspaceOnMetalDebianImage,
+                              RackspaceOnMetalFedoraImage, RackspaceOnMetalUbuntuImage)
+
+
+@attributes([Attribute("image_store", default_factory=list)])
```

Comment:
If this is an `ImageStore`, but it also *has* an `image_store`, what is the difference in role between this object and that attribute?  I think you should add a `:var:` ReST field to the class docstring to document this attribute.

Also, `default_factory` says it's a `list`.  Perhaps it would be better named as `images_list`?